src/qibo/backends/profiles.yml

```diff
@@ -3,19 +3,23 @@ backends:
   - name: qibojit
     class: JITCustomBackend
     from: qibojit.custom_operators
+    minimum_version: 0.0.4.dev0
```

Comment:
Do we need to use "dev0" here and or would just "0.0.4" be sufficient? Same for qibotf. Also, I think the pre-relased version of qibojit and qibotf is "rc0" no "dev0", right?

Reply:
Indeed, I have just fixed the versioning of qibojit and qibotf, given that after 0.0.4rc0 the next natural version is 0.0.4rc1.dev1. I have just included this fix for both repos, and for qibo in this PR.